pyfiglet/__init__.py

```diff
@@ -819,7 +852,11 @@ def main():
         # Set stdout to binary mode
         sys.stdout = sys.stdout.detach()
 
-    sys.stdout.write((r + '\n').encode('UTF-8'))
+    foreground_color = __get_ascii_color(opts.foreground_color, isBackground=False)
+    background_color = __get_ascii_color(opts.background_color, isBackground=True)
+    colors_string = foreground_color + background_color    
+    default_colors = __get_ascii_color('DEFAULT', True) + __get_ascii_color('DEFAULT', False)
```

Comment:
Instead of resetting the code by specifying the colors, better to tell the terminal to reset using `"\033[m"`, see https://en.wikipedia.org/wiki/ANSI_escape_code#CSI_sequences

Reply:
Done!